test/index.js

```diff
@@ -627,10 +627,13 @@ test('non-object target', function (t) {
 	t.end();
 });
 
-test('__proto__ is not merged', function (t) {
+test('__proto__ is merged as an own property', function (t) {
 	var malicious = JSON.parse('{ "fred": 1, "__proto__": { "george": 1 } }');
 	var target = {};
 	extend(true, target, malicious);
 	t.notOk(target.george);
+	t.ok(Object.prototype.hasOwnProperty.call(target, '__proto__'));
+	var name = '__proto__';
+	t.equal(target[name].george, 1);
```

Comment:
`t.deepEqual(target.__proto__, { george: 1 })` would also work

Reply:
For sure.

The reason for the `var name = '__proto__'` gymnastics: the linter discourages both `target.__proto__` (no-proto) and `target['__proto__']` (dot-notation). After some thought, I want to drop the gymnastics and use a comment to disable no-proto on that line.